### DISCUSSION TITLE
Does TypeSafe in Pinia mean that we can ignore TypeScript?

### DISCUSSION THREAD

**Discussion by cleangista** [vuejs/pinia#2166] [Help and Questions] [answered]
Does TypeSafe in Pinia mean that we can ignore TypeScript?

It is stated that TypeScript is for "Static Type Checking". 

If Pinia is Type Safe, is this suggesting that if we need Static Type Checking in our Apps, that Pinia can provide that feature, and we don't need TypeScript for that?

**Comment by posva** [comment, marked as answer]
No, you cannot ignore TypeScript. Pinia uses TypeScript to provide type safety.
